website/docs/sdk-reference/elixir.md

```diff
@@ -382,9 +576,20 @@ Debug level logging helps to inspect how a feature flag was evaluated:
 
 ```bash
 [debug]  Evaluating get_value('isPOCFeatureEnabled').
-[debug]  User object: %ConfigCat.User{country: nil, custom: %{}, email: "configcat@example.com", identifier: "435170f4-8a8b-4b67-a723-505ac7cdea92"}
-[debug]  Evaluating rule: [Email:configcat@example.com] [CONTAINS] [@something.com] => no match
-[debug]  Evaluating rule: [Email:configcat@example.com] [CONTAINS] [@example.com] => match, returning: true
+User object: %ConfigCat.User{country: nil, custom: %{}, email: "configcat@example.com", identifier: "435170f4-8a8b-4b67-a723-505ac7cdea92"}
+Evaluating rule: [Email:configcat@example.com] [CONTAINS] [@something.com] => no match
+Evaluating rule: [Email:configcat@example.com] [CONTAINS] [@example.com] => match, returning: true
+```
+
+The following example shows how to set the _Log Level_ on the internal _ConfigCat_ logger.
+put the following code into your application.ex file and runs it on start:
+
+```elixir
+defp set_config_cat_log_level do
+  :configcat
+  |> Application.spec(:modules)
+  |> Logger.put_module_level(:debug)
+end
```

Comment:
It might be worth mentioning that Elixir 1.13 and later also has `put_application_level` which is simpler. It might also be worth linking to the Elixir docs:

- https://hexdocs.pm/logger/1.15.6/Logger.html#put_module_level/2
- https://hexdocs.pm/logger/1.15.6/Logger.html#put_application_level/2